app/controllers/remote_search_controller.rb

```diff
@@ -1,8 +1,14 @@
 class RemoteSearchController < ApplicationController
   def show
+    @form = SearchForm.new(params[:search_form])
```

Comment:
That makes sense then. As you say, it would be clearer in intent, to move this closer to where it is used.